test/verify/check-connection

```diff
@@ -259,9 +259,9 @@ class TestConnection(MachineCase):
         self.assertIn("Content-Type: text/html\r\n", headers)
         # login.html is not always accessible as a file (e. g. in Atomic), so just assert a reasonable content length
         self.assertIn("Content-Length: ", headers)
-        l = int(headers.split('Content-Length: ', 1)[1].split()[0])
```

Comment:
This commit has the wrong Closes # - also, let's rather add it to the topmost one which enables pycodestyle testing in test-static-code.

Reply:
Done